### PR TITLE
[dashboard] Standardize PendingChangesDropdown component in workspaces list (replaces Modal) and stopped page (unchanged)

### PR DESCRIPTION
**Pull request by jankeromnes**
Align with [spec](https://www.figma.com/file/XQOjjEHO0ALtxdIT9Blzh2/Dark-Theme-%233806) and side effect fix for dark theme:

image (2)

### CODE REVIEW

**Comment by jankeromnes** [comment]
Currently looks like this:

<table><tr><td>
Screenshot 2021-04-27 at 14 03 34
</td><td>
Screenshot 2021-04-27 at 14 03 50
</td></tr></table>

Problems:
- Vertical alignment is off
- Clicking on it doesn't work

**Comment by jankeromnes** [comment]
Now looks like this:

<table><tr><td>
Screenshot 2021-04-27 at 14 25 01
</td><td>
Screenshot 2021-04-27 at 14 25 17
</td></tr><tr><td>
Screenshot 2021-04-27 at 14 25 45
</td><td>
Screenshot 2021-04-27 at 14 25 29
</td></tr></table>

**Comment by jankeromnes** [comment]
@gtsiolis could you please take a look, and make a final decision? 🙏

- Either we go with this PR, replacing the Changes Modal with a dropdown

- Or we temporarily park this PR to keep the Changes Modal, and simply adjust the balloon colors for dark theme, and fix the Modal-induced scrollbar

**Comment by gtsiolis** [comment]
Looking at this now! 👀

**Comment by jankeromnes** [comment]
Probably not critical for the release tomorrow (since dark theme is soft-launch, and there will probably be other small things to fix here and there 😇).

So: no rush 🌴

**Review by gtsiolis — APPROVED** [review]
Thanks for aligning this @jankeromnes! UX looks good! 🌟 

Alright, I think this looks good to merge as is! We could probably add some hover and focus states on this new changes dropdown but this is certainly out of the scope of this PR.

File: components/dashboard/src/workspaces/WorkspaceEntry.tsx [hunk 1]
```diff
@@ -155,36 +133,6 @@ export function WorkspaceEntry({ desc, model, isAdmin, stopWorkspace }: Props) {
     </div>;
 }
 
-export interface PendingChanges {
-    message: string, items: string[]
-}
-
-export function getPendingChanges(wsi?: WorkspaceInstance): PendingChanges[] {
```
Comment (gtsiolis) [inline components/dashboard/src/workspaces/WorkspaceEntry.tsx:162]: **praise**: Deleted code is the best code! 💯

**Comment by gtsiolis** [comment]
> Either we go with this PR, replacing the Changes Modal with a dropdown
> ...

Sounds like a good plane for now! 🗺️ 

---

Posting this here for future reference:

Also, this seems like a good **two-way door decision** which allows us to come back and reverse this change in the future! 🔮

**Comment by jankeromnes** [comment]
Awesome, many thanks for the review! Then let's go forward with unifying this, but I'm more than happy to re-introduce a Changes Modal later. 🚀

**Pull request merged**